src/main/java/com/revature/studyforce/notification/ServletInitializer.java

```diff
@@ -0,0 +1,16 @@
+package com.revature.studyforce.notification;
+
+import com.revature.studyforce.StudyForceApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+/**
+ * Used to configure HATOEOS links in JPA repository
+ * @author Ronald Lopez
+ */
+public class ServletInitializer extends SpringBootServletInitializer {
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
+        return application.sources(StudyForceApplication.class);
+    }
```

Comment:
should be moved to package src/main/java/com/revature/studyforce/configuration
 or considered for deletion.

Reply:
deleted